docs/src/surfaces/drawer/demo/demoDrawer.jsx

```diff
@@ -0,0 +1,618 @@
+import {
+    Drawer,
+    Icon,
+    Typography,
+} from '@saddlebackchurch/react-cm-ui'; // eslint-disable-line import/no-unresolved
+import _ from 'lodash';
+import { connect } from 'react-redux'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
`react-redux` is a proper dependency, isn't it?  Do we need this suppression?

Reply:
Fixed